redis_websocket_api/server.py

```diff
@@ -104,5 +106,9 @@ def listen(self, host, port, channel_names=(), channel_patterns=(), loop=None):
 if __name__ == "__main__":
     import aioredis
 
+    class PublishEverythingHandler(WebsocketHandler):
```

Comment:
What is the purpose of this class? Where is it used?

Reply:
I moved it to `__main__.py` and added a description in the README:

> For a quick test you can run
> 
> ```bash
> python -m redis_websocket_api
> ```
> 
> to start a simple redis websocket api on `ws://localhost:8765`.